Dockerfile

```diff
@@ -68,9 +69,13 @@ RUN pip uninstall -y megatron-core && \
     fi && \
     pip install -e .
 
-WORKDIR /opt
-
-# install the latest NeMo-Aligner
-RUN pip install --no-deps git+https://github.com/NVIDIA/NeMo-Aligner.git@main
+RUN git clone https://github.com/NVIDIA/NeMo-Aligner.git && \
```

Comment:
I would add a comment above `# install the latest NeMo-Aligner`

Reply:
ah right, how's this? 4bac22fdfc57082f0f8bb83c6a80cf8f50f78b2c